test/SourceKit/VariableType/ranged.swift

```diff
@@ -0,0 +1,8 @@
+let x = 1
+let y = "abc"
+let z: String = "def"
+var w = 4
+
+// RUN: %sourcekitd-test -req=collect-var-type -pos=2:1 -length=23 %s -- %s | %FileCheck %s
```

Comment:
I *think* you can also use `-end-pos` here instead of `-length`. That would make the test case more resistent if someone chose to rename one of the variables/types/values